src/lib.rs

```diff
@@ -32,6 +32,11 @@ use std::{sync::Arc, time::Duration};
 use rand::prelude::*;
 use tracing::{debug, info, info_span};
 
+#[cfg(all(not(target_os = "emscripten"), target_pointer_width = "64"))]
+type SmallRng = rand_pcg::Pcg64Mcg;
+#[cfg(not(all(not(target_os = "emscripten"), target_pointer_width = "64")))]
```

Comment:
I'm not sure I follow why the `#[cfg]` is needed here — why not use the same one on all targets?

Reply:
I just copied the code from rand 0.7

Reply:
Ah, I see! I think I'd prefer to go the `u64` path and stick with `rand`'s `SmallRng`. Given that we'll probably end up with a breaking change from https://github.com/jonhoo/bustle/issues/7, I'm okay with breaking on this now.